### PR TITLE
Fix toggle_location_services not working on android 9.0 devices

### PR DESCRIPTION
**Pull request by gillett-hernandez**
unfortunately i haven't been able to test this end to end
i did test by manually inputting the adb keycode sequence and it toggled the location

this potentially fixes https://github.com/appium/appium/issues/13755

### CODE REVIEW

File: lib/commands/network.js [hunk 1]
```diff
@@ -168,7 +168,7 @@ commands.toggleLocationServices = async function toggleLocationServices () {
     if (api === 16) {
       // This version of Android has a "parent" button in its action bar
       seq.push(20); // down
```
Comment (mykola-mokhnach) [inline lib/commands/network.js]: could we also put these magic numbers to constants, so then the comments, like `up, up` are not needed?

**Comment by mykola-mokhnach** [comment]
@gillett-hernandez Is there any chance to finish the PR?

**Comment by gillett-hernandez** [comment]
@mykola-mokhnach i gave it a shot but i'm a bit rusty on my javascript. let me know if what i wrote as a refactor is not the right way to do it.

File: lib/commands/network.js [hunk 2]
```diff
@@ -152,7 +152,12 @@ commands.getGeoLocation = async function getGeoLocation () {
     altitude: parseFloat(altitude) || GEO_EPSILON,
   };
 };
-
+// https://developer.android.com/reference/android/view/KeyEvent.html#KEYCODE_DPAD_CENTER
+// in the android docs, this is how the keycodes are defined
+let up = 19;
```
Comment (mykola-mokhnach) [inline lib/commands/network.js]: good, although can be improved:

```
// Move the declaration to the top of the module
const KeyCode = {
  UP: 19,
  DOWN: 20,
  ...
}
...

let seq = [KeyCode.UP, KeyCode.UP];
...
...
```

**Comment by gillett-hernandez** [comment]
@mykola-mokhnach updated. CI is running now. if it looks good to you, i'd love for it to be merged.

**Comment by mykola-mokhnach** [comment]
https://travis-ci.org/appium/appium-android-driver/jobs/634469166#L3293 Please add the missing semicolon to make tests pass

**Pull request merged**